docusaurus-docs/community/design/user-personas.md

```diff
@@ -0,0 +1,191 @@
+---
+description: Describes current and potential conda-store users`
+---
+
+# User groups and personas
+
+The personas used for informing user research studies and future improvements to conda-store.
+
+## User groups 🏘️
+
+At a high-level, there are two key user groups:
+
+- **General user:** Create, edit, use environments for their day-to-day work
+- **Admin-type user:** In addition to the general use-case mentioned above, admin-type users also **setup** conda-store for their team, create various namespaces, add/update users and their permissions, set default packages/versions for their team, etc.
+
+## User personas 🙋🏽‍♀️
+
+### User: Alia
+
+| Role | Reports to | User group|
+|-------|-------|-------|
+Staff Data Scientist | Head of Data Science | General |
+
+#### User story
+
+I want to quickly create a stable and reproducible environment for analyzing data and creating dashboards, that I can share with my team to verify and build on my work.
+
+#### Tools they need to do their job
+
+- Libraries - data reader/writer (arrow), data processing (pandas), numerical computing (numpy), ML (xgboost / pytorch), dataviz and dashboarding (mpl / streamlit), job schedulers (Airflow)
+- IDE - JupyterLab/Notebook (hence, extensions)
+- Platform - Local computer, org JupyterHub on Cloud (Nebari?)
+- Misc - Git, GitHub, conda
+
+#### Journey with conda-store
+
+- Discovery:
+    - Team already uses it
+    - Team leader sets it up for everyone
+- Onboarding:
+    - A team member gives them a walkthrough
+    - Documented Tutorials
+    - Intuitiveness of the UI
+- General Use:
+    - Env creation from GUI, YAML, Lockfile
+    - Version control
+    - Using the env in Jupyter Notebook (local / cloud)
+    - Ability to use any package/version out there
+    - Access to special conda channels
+    - Envs for GPU-powered data analysis and modelling
+- Collaboration:
+    - Share env with colleagues with a URL (Part of 1-2 shared namespaces)
+    - Fork/Copy envs between namespaces
+    - Download artifacts like docker images for use in other (prod?) machines
+- Troubleshooting:
+    - Understand errors through the error message displayed
+    - Look at conda-store’s documentation & GH issues/discussions, search for the error
+    - Contact team-members or internal support
+    - Open an issue/discussion on GH
+
+#### Pain points or biggest challenges
+
+- Envs need to be compliant with company standards
+- Build envs with libraries from internal mirrors
```

Comment:
This is in contrast with "ability to use any package/version out there" https://github.com/conda-incubator/conda-store/pull/773/files#diff-99d5523c0e67c61630b4cfb694883c9282df0074dcd29f29078be74bdb1efc2fR48

Reply:
I don't think so? We meant this as: internal mirrors are one of the various places we'd want to install packages from, right?